### ISSUE TITLE
Use a table component for file lists

### ISSUE THREAD

**Issue by HEnquist** [HEnquist/camillagui#91]
Replace the simple list in the file chooser popups with something better. This should enable filtering and sorting.

**Comment by JWahle** [comment]
Makes sense, I'll try an implementation, unless you did something here already.

**Comment by HEnquist** [comment]
I haven't done anything except for a quick search for table components for react, just to see if the idea had any future. I found a several but didn't look closely at any of them.

**Comment by HEnquist** [comment]
The motivation for this is that implementing things like sorting and filtering of lists feels like a waste of time if there is a nice component that can be imported. That should save quite some time, and could give a lot of other functionality for free such as pagination and free-text filters.
